Core/Src/InitializationTask.c

```diff
@@ -0,0 +1,64 @@
+#include "Tasks.h"
+#include "config.h"
+#include "cmsis_os.h"
+#include "fatfs.h"
+#include "lwip.h"
+#include "IMU.h"
+#include "SDCard.h"
+#include "Tasks.h"
+#include "radio.h"
+#include "CANBus.h"
+
+#if CAN_LOOPBACK
+    #define CURR_CAN_MODE       CAN_MODE_LOOPBACK
+#else
+    #define CURR_CAN_MODE       CAN_MODE_NORMAL
+#endif
+
+osThreadId_t DataLoggingTaskHandle;
+osThreadAttr_t DataLoggingTask_attributes = {
+  .name = "Data Logging Task",
+  .priority = (osPriority_t) osPriorityNormal,
+  .stack_size = 1024
+};
+osThreadId_t DataReadingTaskHandle;
+osThreadAttr_t DataReadingTask_attributes = {
+  .name = "Data Reading Task",
+  .priority = (osPriority_t) osPriorityNormal,
+  .stack_size = 1024
+};
+osThreadId_t BroadcastingTaskHandle;
+osThreadAttr_t BroadcastingTask_attributes = {
+  .name = "Broadcasting Task",
+  .priority = (osPriority_t) osPriorityBelowNormal,
```

Comment:
What was the reason for having this be lower than the rest?

Reply:
I forgot lol